test/tests/functional/pbs_ralter.py

```diff
@@ -391,11 +390,24 @@ def alter_a_reservation(self, r, start, end, shift,
                 msg = "Resv;" + r + ";Reservation alter confirmed"
             else:
                 msg = "Resv;" + r + ";Reservation alter denied"
-
-            lines = self.server.log_match(msg, n='ALL', allmatch=True,
-                                          interval=2, max_attempts=30)
-            self.assertEqual(len(lines), sequence)
-
+            attempts = 1
+            interval = 0.5
+            max_attempts = 30
+            while attempts <= max_attempts:
+                lines = self.server.log_match(msg, n='ALL', allmatch=True,
+                                              interval=1, max_attempts=30)
+                info_msg = "log_match: searching " + \
+                    str(sequence) + " sequence of message: " + \
+                    msg + ": Got: " + str(len(lines))
+                if len(lines) == sequence:
+                    self.assertEqual(len(lines), sequence, info_msg)
```

Comment:
This assert will always be true.  You're checking for the same equality in the if above it.

Reply:
Changed to info log message.